packages/wix-ui-tpa/src/components/MobileDrawer/MobileDrawer.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import { classes } from './MobileDrawer.st.css';
```

Comment:
```suggestion
import { st, classes } from './MobileDrawer.st.css';
```

You should use the st function